### ISSUE TITLE
Exception and blue screen on enabling multi-user whiteboard

### ISSUE THREAD

**Issue by fcecagno** [bigbluebutton/bigbluebutton#6812]
This is the exception printed on client console log:

```
Exception { name: "NS_ERROR_FAILURE", message: "", result: 2147500037, filename: "https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true", lineNumber: 125, columnNumber: 0, data: null, stack: "calculateCursorLabelBoxDimensions@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:125:182995
componentDidMount@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:125:182290
Xi@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:9:137191
Yi@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:9:133143
qi@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:9:132488
Ui@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:9:131509
hi@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:9:130407
enqueueForceUpdate@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:9:95959
react.production.min.js/x.prototype.forceUpdate@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:9:35140
calculateData/this.computation</<@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:85:1805
_compute@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:47:2308
_recompute@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:47:2471
tracker.js/s._runFlush@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:47:3483
a@https://dev22.bigbluebutton.org/html5client/f68273cfda959e7c71e280b80efd1a2608754d1f.js?meteor_js_resource=true:1:3032
" }
```

It looks like it has something to do with moving mouse near the edge of the presentation while multi-user whiteboard is enabled.

**Comment by capilkey** [comment]
Could be related to this other issue with the cursor and multi-user whiteboard, https://github.com/bigbluebutton/bigbluebutton/issues/6613.

**Comment by antobinary** [comment]
Combining into #6857

**Comment by jfsiebel** [comment]
This can be merged with #6857 or vice versa, it's the same problem in both

**Comment by antobinary** [comment]
Combining into #6857
